include/klee/Config/config.h

```diff
@@ -13,4 +13,4 @@
 // limitations under the License.
 
 #define LLVM_VERSION_MAJOR 3
-#define LLVM_VERSION_MINOR 6
```

Comment:
The souper trunk is already at LLVM_VERSION_MINOR 7 since the last bump. Raimonads could you make sure that your patch here is against the latest master?